scripts/dora-metrics/generate_metrics.sh

```
@@ -0,0 +1,332 @@
+
+    local gh_cmd="$GH_BIN run list --workflow=$WORKFLOW_NAME --limit=$LIMIT"
+
+    if [[ -n "$CREATED_AFTER" ]]; then
+        gh_cmd="$gh_cmd --created >=$CREATED_AFTER"
+    fi
+
+    if [[ -n "$CREATED_BEFORE" ]]; then
+        gh_cmd="$gh_cmd --created <=$CREATED_BEFORE"
+    fi
+
+    local runs=$(eval "$gh_cmd --json databaseId,status,conclusion,createdAt,updatedAt,headSha,event --jq '.[] | {
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)